test/users.test.js

```diff
@@ -135,4 +136,89 @@ describe('users', () => {
         .then(() => done());
     });
   });
+  describe('/users/sessions POST', () => {
+    it('should be successful', done => {
+      chai
+        .request(server)
+        .post('/users/sessions')
+        .send({
+          password: 'password1234',
+          email: 'joe.doe@wolox.com.ar'
+        })
+        .then(res => {
+          res.should.have.status(200);
+          res.should.be.json;
+          res.headers.should.have.property(sessionManager.HEADER_NAME);
+          sessionManager
+            .decode(res.headers[sessionManager.HEADER_NAME])
+            .should.be.equal('joe.doe@wolox.com.ar');
+          dictum.chai(res);
+        })
+        .then(() => done());
+    });
+    it('should fail because mail is invalid', done => {
+      chai
+        .request(server)
+        .post('/users/sessions')
+        .send({
+          password: 'password1234',
+          email: 'joe.doe@other.com'
+        })
+        .then(res => {
+          res.should.have.status(400);
+          res.should.be.json;
+          res.body.should.have.property('message');
```

Comment:
why do you don't check message text?

Reply:
I forget error messages, my bad. Resolved!